lib/routes/cline/blog.ts

```diff
@@ -0,0 +1,187 @@
+        logger.error('Error parsing blog data from script:', error);
+    }
+
+    return [];
+}
+
+// Fetch article detail content
+async function fetchArticleDetail(article: Article): Promise<Article> {
+    try {
+        const detailResponse = await got({
+            method: 'get',
+            url: article.link,
+        });
+
+        const content = load(detailResponse.data);
+        const articleContent = content('main article, .blog-post-content, .article-content');
+
+        if (articleContent.length > 0) {
+            let contentHtml = articleContent.html() || '';
+            contentHtml = contentHtml.replaceAll('src="/', `src="${rootUrl}/`).replaceAll('href="/', `href="${rootUrl}/`);
+
+            return {
+                ...article,
+                description: contentHtml,
+            };
+        }
+    } catch {
+        // If fetching detail fails, return original article
+    }
+
+    return article;
+}
+
+async function handler() {
+    // Get blog homepage
+    const response = await got({
+        method: 'get',
+        url: blogUrl,
+    });
+
+    const $ = load(response.data);
+
+    // First try to extract articles from script
+    let articles: Article[] = [];
+
+    const scriptContents = $('script')
+        .toArray()
+        .map((script) => $(script).html());
+
+    // Iterate through all script contents to extract article information
+    for (const content of scriptContents) {
+        if (content) {
+            const extractedArticles = extractArticlesFromScript(content);
+            if (extractedArticles.length > 0) {
+                articles = extractedArticles;
+                break;
+            }
+        }
+    }
+
+    // If extraction from script fails, try extracting from DOM
+    if (articles.length === 0) {
+        articles = extractArticlesFromDOM($);
+    }
+
+    // If still no articles, provide a fallback content
+    if (articles.length === 0) {
+        articles = [
+            {
+                title: 'Cline Official Blog',
+                link: blogUrl,
+                description: 'Unable to fetch Cline blog articles. Please visit the official blog directly.',
+            },
+        ];
+    }
+
+    // Fetch article details
+    const items = await Promise.all(articles.map((article) => cache.tryGet(article.link, () => fetchArticleDetail(article))));
+
+    return {
+        title: 'Cline Official Blog',
+        link: blogUrl,
+        item: items,
+        description: 'Cline Official Blog - AI Coding Assistant',
+    };
+}
+
+export const route: Route = {
+    path: '/blog',
+    categories: ['blog'],
+    example: '/cline/blog',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: 'Blog',
+    maintainers: ['GitHub-Copilot'],
+    description: `Cline Official Blog articles`,
```

Comment:
```suggestion
    description: 'Cline Official Blog articles',
```
Only use template literals when necessary